kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flights/connection/Connection.java

```diff
@@ -0,0 +1,55 @@
+package com.kodilla.good.patterns.flights.connection;
+
+import com.kodilla.good.patterns.flights.flight.Flight;
+import com.kodilla.good.patterns.flights.flight.FlightVia;
+import com.kodilla.good.patterns.flights.flight.Town;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Connection implements Connectable {
+
+    @Override
+    public Set<Flight> findFlightFrom(Set<Flight> flights, String destination) {
```

Comment:
W każdej metodzie przekazujesz parametr Set<Flight> - co byś powiedział na to, aby obiekt klasy `Connection`  zawierał pole Set<Flight> flights, ktore ustawisz w konstruktorze?